integration-tests/support/testcontainers-support/src/main/java/org/apache/camel/quarkus/testcontainers/ContainerUtil.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.quarkus.testcontainers;
+
+import java.util.concurrent.TimeUnit;
+
+import org.testcontainers.containers.GenericContainer;
+
+public class ContainerUtil {
+    /**
+     * Wait for the container to be in running state
+     *
+     * @param container the container to wait for
+     */
+    public static void waitForInitialization(GenericContainer container) {
+        for (int retries = 0; retries < 5; retries++) {
+            if (container.isRunning()) {
```

Comment:
FYI `container.start()` will throw an error if the container is not running, so I assume this check always pass and this helper isn't needed. Let us know otherwise, there must be a bug then ;)

Reply:
will have a look after we get this merged, thx for the support

Reply:
> FYI `container.start()` will throw an error if the container is not running, so I assume this check always pass and this helper isn't needed. Let us know otherwise, there must be a bug then ;)

Really nice to have you here @bsideup , thanks for helping